SonarQube.Client/Api/V8_1/GetHotspotRequest.cs

```diff
@@ -46,7 +47,7 @@ protected override SonarQubeHotspot ParseResponse(string response)
                 serverResponse.Project.Key,
                 serverResponse.Project.Name,
                 serverResponse.Component.Key,
-                serverResponse.Component.Path,
+                FilePathNormalizer.NormalizeSonarQubePath(serverResponse.Component.Path),
```

Comment:
This doesn't feel right; we shouldn't be using the same property name (`ComponentPath`) as the server API now we're transforming the data. We should rename the property to something else e.g. 'RelativeFilePath', `ServerFilePath`, with an appropriate doc comment on the property.

It's not a problem with SonarQubeIssue class since the property is already called `FilePath`.